source/activity_api.rst

```diff
@@ -0,0 +1,140 @@
+************************************
+Activity API
+************************************
+
+1. Get Activity List
+====================
+| URL endpoint: https://stamps.co.id/api/v2/activities/
+| Allowed method: GET
+| Requires authentication: Yes
+
+
+A. Request
+----------
+
+You can query a user's activity list by calling the API with these parameters.
+
+=================== =========== =======================
+Parameter           Required    Description
+=================== =========== =======================
+user                Yes         Email address / Member ID indicating customer
+token               Yes         Merchant's Authentication token
+older_than          No          Activity ID. 50 activities will be returned that are older than this ID. If missing, will return latest 50 activities
```

Comment:
`Activity ID. 50 activities will be returned that are older than this ID. If not provided, will return the latest 50 activities.`